kit/ChildSession.cpp

```diff
@@ -1500,7 +1501,17 @@ bool ChildSession::keyEvent(const StringVector& tokens,
 
     getLOKitDocument()->setView(_viewId);
     if (target == LokEventTargetEnum::Document)
+    {
+        // Check override mode
+        if (type == LOK_KEYEVENT_KEYINPUT && charcode == 0 && keycode == KEY_INSERT &&
+            !config::getBool("overwrite_mode[@enable]", false)) // Problem: it always returns the def value that given as second parameter.
```

Comment:
Because the XPath syntax is wrong. Use overwrite_mode.enable.
overwrite_mode[@enable] would match <overwrite_mode enable="true"/>

Reply:
I thought i tried that syntax too. It works now. Thank you Andras o/